.github/ISSUE_TEMPLATE/b-bug-report.md

```diff
@@ -0,0 +1,22 @@
+---
+name: "🐞 Bug Report"
+about: Bug??? Han, bring me a flamethrower 🚒.
+
+---
+
+## 🐞 Bug Description
+<!-- Short and clear how the bug looks like -->
+<!-- It could be a short converstation with the user -->
+
+## Context
+<!-- Describe the platform and Toggl Version. It's super easy for us to trace back those monter-->
+
+Platform: macOS/Linux/Window
+OS Version: macOS 10.14 or Window 10.1
+Toggl Version: 7.4.253
+
+## Expected behavior
+<!-- A clear and concise description of what you expected to happen -->
+
+## To reproduce
+<!-- Clear steps to reproduce the issue -->
```

Comment:
I would make this comment multiline and add a numbered list template so it's clear we want the order of the steps we should take to reproduce this.
Also, "To reproduce" should be something along the lines of "Can this behavior be reproduced every time?" and then let the user describe the steps, if yes.

Reply:
Good point. I will update it 👍